webpush_channels/tests/test_views_channel.py

```diff
@@ -20,6 +23,28 @@ def test_registration_can_be_remove_for_a_user(self):
         resp = self.app.get(self.channel_url, headers=self.headers, status=200)
         assert resp.json['data']['registrations'] == 0
 
+    def test_remove_non_existent_registration(self):
+        self.assertRaises(self.app.delete(self.channel_registration_url,
```

Comment:
It could be returning a 404 but also accepting the removal request and just making sure the registration is been deleted.

The analogy I make is: when you switch the light on and it is already on it doesn't tell you "error: lighting a light already on doesn't exists"